dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -75,6 +77,11 @@ export const htmlPageTemplate = (props: WebProps | AppProps): string => {
 	 */
 	const windowGuardian = escapeData(JSON.stringify(guardian));
 
+	if (config.renderingTarget === 'Apps') {
+		config.darkModeAvailable;
+		config.updateLogoAdPartnerSwitch;
+	}
```

Comment:
Was the intention to assign values here?

Reply:
haha no just testing values in the IDE!